worker-core/src/main/java/com/hpe/caf/worker/core/WorkerCore.java

```diff
@@ -486,6 +484,30 @@ public void discard(String queueMsgId)
             stats.incrementTasksDiscarded();
         }
 
+        @Override
+        public void reportUpdate(final String queueMsgId, final String queue, final TaskMessage responseMessage)
```

Comment:
`responseMessage` is a confusing variable name as this is not the response.

Reply:
I agree. I originally had 'reportUpdateMessage' and changed it. Will revert back.